great_expectations/expectations/core/expect_table_row_count_to_be_between.py

```diff
@@ -91,7 +91,8 @@ class ExpectTableRowCountToBeBetween(TableExpectation):
             "round_decimals": 0,
         },
         rules={
-            "row_count_range_rule": {
+            "default_:w"
```

Comment:
```suggestion
```

Reply:
@NathanFarmer thank you for catch this typo!